service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/ManageLegalRepresentativesValidator.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.fpl.service.validators;
+
+import com.google.common.collect.Streams;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.util.Lists;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.LegalRepresentative;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.OrganisationService;
+
+import java.text.MessageFormat;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ManageLegalRepresentativesValidator {
+
+    private static final String VALIDATION_SUFFIX = "for Legal representative";
+
+    private final OrganisationService organisationService;
+
+    public List<String> validate(List<Element<LegalRepresentative>> legalRepresentatives) {
+        return Streams.mapWithIndex(
+            Optional.ofNullable(legalRepresentatives).orElse(Lists.emptyList()).stream(),
+            (legalRepresentativeElement, idx) ->
+                performBasicValidation(legalRepresentativeElement.getValue(), idx, legalRepresentatives.size())
+        ).flatMap(Collection::stream).collect(Collectors.toList());
+    }
+
+    private List<String> performBasicValidation(LegalRepresentative legalRepresentative, long currentRepresentativeIdx,
+                                                int sizeOfRepresentatives) {
+        List<String> validationErrors = Lists.newArrayList();
```

Comment:
Using `new ArrayList<>()` would be nicer here, then you aren't using the testing library for main code. Maybe we need to reduce the scope of the library in the `build.gradle`.

Reply:
missed import